### PR TITLE
Overview on DAS architecture and components

### PR DESCRIPTION
**Pull request by andre-senna**

### CODE REVIEW

File: docs/DAS-Overview.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+pattern matching. When making a query, the user can specify whether only local
+atoms should be considered or whether atoms in remote DASs should be searched
+as well. If that's the case, the Query Engine connects to the remote OpenFaaS
+servers to make the queries in the remote DASs and return a answer which is a
+proper combination of local and remote information. For instance, if there're
+different versions of the same atom in local and one of the remote DASs, the
+local version is returned.
+
+Both engines use the __Cache__ in order to make queries involving a remote DAS
+faster. The DAS' cache is not exactly like a traditional memory or disc cache,
```
Comment (marcocapozzoli) [inline docs/DAS-Overview.md:86]: Remove apostrophe after `DAS`

File: docs/DAS-Overview.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+2. To instantiate a DAS in a Python program which can store a smaller local
+knowledge base and can, optionally, connect to one or more remote DAS servers,
+exposing their contents to the local program. In this case, the local knowledge
+base can store its contents in RAM or can use a DB backend to persist it.
+
+<p align="center">
+<img src="assets/components.png" width="900"/>
+</p>
+
+Components in the DAs architecture are designed to provide the same
```
Comment (marcocapozzoli) [inline docs/DAS-Overview.md:62]: Capitalize the letter 'S'

File: docs/DAS-Overview.md (hunk 1 above)
Comment (andre-senna) [reply docs/DAS-Overview.md:86]: This is grammar. `DAS' cache` means `Cache of DAS`

File: docs/DAS-Overview.md (hunk 2 above)
Comment (andre-senna) [reply docs/DAS-Overview.md:62]: Done

**Review by marcocapozzoli — APPROVED** [review]

**Review by levisingularity — APPROVED** [review]

**Pull request merged**